README.md

```diff
@@ -36,7 +38,7 @@ Includes -
 * Consul, running in server mode (Optional - configure via ```consul_enable:``` param in ```hieradata/common.yml```  )
 
     $ cd standalone
-    $ vagrant up
+    $ vagrant up (ubuntu/centos)
```

Comment:
will add some about supporting operating systems.

Will do a seperate pull request for the cluster.